misc/language-tour-guide/src/components/navigation/ScrollToHash.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router";
+
+export function ScrollToHash() {
+  const { hash } = useLocation();
+
+  useEffect(() => {
+    if (hash) {
+      const id = hash.slice(1);
+      const el = document.getElementById(id);
+      if (el) {
+        setTimeout(() => {
+          el.scrollIntoView();
+        }, 100);
+      }
+    } else {
+      const descriptionSection = document.getElementById("description");
+      if (!descriptionSection) {
+        return;
+      }
```

Comment:
In block above you do the logic inside if and in this block you reverse it. Please choose the same approach in both cases.